frontend/tests/controllers-test/SubmissionFilesCtrl.test.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+describe('Unit tests for Submission Files Controller', function () {
```

Comment:
`Unit tests for Submission Files Controller` -> `Unit tests for submission files controller`

Reply:
@RishabhJain2018 done